Sources/MapboxNavigation/MapboxSpeechSynthesizer.swift

```diff
@@ -69,6 +72,12 @@ open class MapboxSpeechSynthesizer: NSObject, SpeechSynthesizing {
         self.remoteSpeechSynthesizer = SpeechSynthesizer(accessToken: accessToken, host: hostString)
     }
     
+    /// Creates new `MapboxSpeechSynthesizer` with provided `SpeechSynthesizer` instance for converting text to audio.
+    public init(speechSynthesizer: SpeechSynthesizer) {
+        self.cache = DataCache()
+        self.remoteSpeechSynthesizer = speechSynthesizer
```

Comment:
It’s kind of confusing to have MapboxSpeechSynthesizer conform to SpeechSynthesiz**ing** but encapsulate a SpeechSynthesiz**er** from MapboxDirections. If that’s unavoidable, maybe we can rename the argument to `remoteSpeechSynthesizer` to clarify the distinction.